src/main/java/org/springframework/data/redis/connection/lettuce/LettuceReactiveZSetCommands.java

```diff
@@ -189,28 +190,28 @@
 				if (command.isWithScores()) {
 
 					result = cmd
-							.zrangeWithScores(command.getKey(), command.getRange().getLowerBound().getValue().orElse(null),
-									command.getRange().getUpperBound().getValue().orElse(null))
+							.zrangeWithScores(command.getKey(), command.getRange().getLowerBound().getValue().orElse(0L),
+									command.getRange().getUpperBound().getValue().orElse(Long.MAX_VALUE))
 							.map(sc -> (Tuple) new DefaultTuple(getBytes(sc), sc.getScore()));
 				} else {
 
 					result = cmd
-							.zrange(command.getKey(), command.getRange().getLowerBound().getValue().orElse(null),
-									command.getRange().getUpperBound().getValue().orElse(null))
+							.zrange(command.getKey(), command.getRange().getLowerBound().getValue().orElse(0L),
+									command.getRange().getUpperBound().getValue().orElse(Long.MAX_VALUE))
 							.map(value -> (Tuple) new DefaultTuple(ByteUtils.getBytes(value), Double.NaN));
 				}
 			} else {
 				if (command.isWithScores()) {
 
 					result = cmd
-							.zrevrangeWithScores(command.getKey(), command.getRange().getLowerBound().getValue().orElse(null),
-									command.getRange().getUpperBound().getValue().orElse(null))
+							.zrevrangeWithScores(command.getKey(), command.getRange().getLowerBound().getValue().orElse(0L),
```

Comment:
I do think that the -inf at https://redis.io/commands/zrevrangebyscore should be translated to Long.MIN_VALUE here.